indico/modules/events/registration/templates/management/_invitation_list.html

```diff
@@ -36,7 +36,17 @@
                         {{ invitation.affiliation }}
                     </td>
                     <td class="actions">
-                        <a class="icon-remove js-delete-invitation"
+                        {% if invitation.state == invitation.state.pending %}
+                        <a class="icon-disable js-invitation-action"
+                           title="{% trans %}Mark as declined{% endtrans %}"
+                           data-href="{{ url_for('.mark_invitation_as_declined', invitation) }}"
+                           data-method="POST"
+                           data-title="{% trans %}Mark invitation as declined{% endtrans %}"
+                           data-confirm="{% trans %}Do you really want to mark this invitation as declined by the invitee?{% endtrans %}"
+                           ></a>
```

Comment:
probably for readability ;) but i'm also not a huge fan of having a `>` as the first thing in a line